go/exorelay/exorelay_test.go

```diff
@@ -126,6 +164,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
+		mapping[messageName] = func(payload map[string]interface{}) {
+			handlerCalled = true
+			savedPayload = payload
+		}
+		exoInstance.BindMessageHandlers(mapping)
```

Comment:
Looking at this again, what is the motivation for having to provide all message handlers all at once here? Maps are mutable, why can't we provide an API that allows registering one handler at a time?

```go
exoInstance.bindMessageHandler(messageName, func(payload map[string]interface{}) {
  handlerCalled = true
  savedPayload = payload
}